pkg/micropython/patches/0001-Ensure-mp-int-and-uint-have-pointer-size.patch

```diff
@@ -0,0 +1,35 @@
+From 9333396d68f328b84a973757fd36522aec74a0f5 Mon Sep 17 00:00:00 2001
+From: Leon Hielscher <hielscher@fzi.de>
+Date: Tue, 11 Aug 2020 23:58:10 +0200
+Subject: [PATCH 1/2] Ensure mp int and uint have pointer size.
+
+---
+ ports/riot/mpconfigport.h | 5 +++--
+ 1 file changed, 3 insertions(+), 2 deletions(-)
+
+diff --git a/ports/riot/mpconfigport.h b/ports/riot/mpconfigport.h
+index d02e96653..0387ea597 100644
+--- a/ports/riot/mpconfigport.h
++++ b/ports/riot/mpconfigport.h
+@@ -24,6 +24,7 @@
+  * THE SOFTWARE.
+  */
+ #include <stdlib.h>
++#include <stdint.h>
+ 
+ #include "irq.h"
+ 
+@@ -96,8 +97,8 @@
+ 
+ #define MICROPY_MODULE_FROZEN_STR       (1)
+ 
+-typedef int mp_int_t;       // must be pointer size
+-typedef unsigned mp_uint_t; // must be pointer size
++typedef intptr_t mp_int_t;       // must be pointer size
++typedef uintptr_t mp_uint_t; // must be pointer size
```

Comment:
Since we are using [our own fork](https://github.com/kaspar030/micropython) anyway, you might as well open the PR there.

Reply:
True - but actually unrelated to this PR, isn't it?

Reply:
Made a PR in the fork.

Reply:
PR is merged, waiting for #20208 to get merged before we can drop this patch.